oggm/cfg.py

```diff
@@ -857,9 +857,19 @@ def set_manager(manager):
         new_dict = manager.dict()
         new_dict.update(DEM_SOURCE_TABLE)
         DEM_SOURCE_TABLE = new_dict
+
+        new_dict = manager.dict()
+        new_dict.update(DATA)
+        DATA = new_dict
+
+        new_dict = manager.dict()
+        new_dict.update(LRUHANDLERS)
+        LRUHANDLERS = new_dict
     else:
         DL_VERIFIED = dict(DL_VERIFIED)
         DEM_SOURCE_TABLE = dict(DEM_SOURCE_TABLE)
+        DATA = dict(DATA)
+        LRUHANDLERS = dict(LRUHANDLERS)
```

Comment:
@TimoRoth what was the purpose of this re-cast to a dict? They all are dict per definition....

Reply:
Ah I think I know.

Reply:
To get rid of the manager, and only keep the data.